keyboards/planck/keymaps/vaire/config.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you remove the first 4 lines here (up to the include, but leave the `#define MUSIC_MASK`), and replace it with `#pragma once`, and remove the endif at the end?